echo10-construction/src/main.py

```diff
@@ -91,7 +91,7 @@ def get_granule_data(inputs, config):
         'beginning_date_time': granule_metadata['sensing_start'],
         'ending_date_time': granule_metadata['sensing_stop'],
         'orbits': granule_metadata['orbit_number'],
-        'platforms': [p.upper() for p in set(granule_metadata['platform'])],
+        'platforms': [p.upper() for p in granule_metadata['platform']],
```

Comment:
Let's double-check that we don't need to worry about duplicate values by getting rid of `set`.